src/components/Modal/SbModal.vue

```diff
@@ -151,6 +151,9 @@ export default {
 
   created() {
     this.$_createPortalInstance()
+    if (this.isOpen) {
+      this.handleCloseModalByPressingEsc()
+    }
```

Comment:
I added this logic here because, currently, when we use the component with `<SbModal is-open />` (i.e: with the `isOpen` prop `true` by default). 
The `handleCloseModalByPressingEsc` is never called because it is called in the `isOpen` watcher which triggers the `handleOpenModal` method. And the modal should not be already open for the event listener to be called.